### PR TITLE
✏️ Rename Toc -> TOC

### PR DESCRIPTION
**Pull request by agoose77**
This is a clean-up PR, before we start doing surgery on TOC handling.

### CODE REVIEW

File: .changeset/wet-beds-change.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+"myst-common": minor
+"myst-cli": minor
```
Comment (fwkoch) [inline .changeset/wet-beds-change.md:3]: I think this is the most notable part of this PR - making a breaking change for some naming consistency.

**Review by fwkoch — APPROVED** [review]
I don't feel super strongly about this - the naming consistency is nice; the breaking change is not so nice. However, if we are making these changes _sometime_, sooner is better than later!

**Comment by agoose77** [comment]
Thanks Franklin! I'm taking that as merge approval :D

**Comment by rowanc1** [comment]
@agoose77 bug I found:

```
myst init --write-toc
```

no longer works.

Will make a patch.

**Pull request merged**